docs/install.rst

```diff
@@ -18,39 +18,71 @@ Requirements
 PlasmaPy requires Python version 3.7 or newer.
 PlasmaPy requires the following packages for installation:
 
-- `NumPy <http://www.numpy.org/>`_ 1.18.1 or newer
-- `SciPy <https://www.scipy.org/>`_ 1.2 or newer
-- `Astropy <http://www.astropy.org/>`_ 4.0 or newer
+- `NumPy <https://www.numpy.org/>`_ — 1.18.1 or newer
+- `SciPy <https://www.scipy.org/>`_ — 1.2 or newer
+- `Astropy <https://www.astropy.org/>`_ — 4.0 or newer
+- `pandas <https://pandas.pydata.org/>`_ — 1.0 or newer
+- `xarray <http://xarray.pydata.org>`_ — above 0.14 but less than 0.16
+- `tqdm <https://tqdm.github.io/>`_ — 4.56 or newer
+- `colorama <https://pypi.org/project/colorama/>`_ — 0.3 or newer
+- `cached_property <https://pypi.org/project/cached-property/>`_ — 1.5.2 or newer
 
 PlasmaPy also depends on the following packages for optional features:
 
-- `matplotlib <https://matplotlib.org/>`_ 2.0 or newer
-- `h5py <https://www.h5py.org/>`_ 2.8 or newer
-- `mpmath <http://mpmath.org/>`_ 1.0 or newer
-- `lmfit <https://lmfit.github.io/lmfit-py/>`_ 1.0.1 or newer
+- `h5py <https://www.h5py.org/>`_ — 2.8 or newer
+- `lmfit <https://lmfit.github.io/lmfit-py/>`_ — 1.0.1 or newer
+- `matplotlib <https://matplotlib.org/>`_ — 2.0 or newer
+- `mpmath <https:htt//mpmath.org/>`_ — 1.0 or newer
```

Comment:
I've been wondering if we should include requirements information here.  My reasons for dropping it are:

 - This information is not necessary if you're installing via pip or conda, including for installing the development version.
 - People who do need the information can go directly to `requirements.txt`.
 - This list is prone to becoming out-of-date, for example if we update `requirements.txt` but forget to update this list.  

An alternative would be to refer users to `requirements.txt` in the section about installing the development version.

I'm not planning to drop it in this PR...just wanted to bring up what I was thinking.

Reply:
Erik's clean up requirements pr would also help here, probably. 

I'd prefer keeping it, but adding a note about where to find the current requirements.

Reply:
> If you're writing a sentence like this, it should be a red flag that multiple PRs would be helpful :D

Yep...in retrospect this is exactly right.  Small, frequent pull requests are better than monolithic pull requests.  Sigh!